Task4

```diff
@@ -0,0 +1 @@
+SELECT age, COUNT(*) AS users FROM users GROUP BY age ORDER BY age asc, users desc;
```

Comment:
ordering order should be swapped here:
`age asc, users desc` -> `users desc, age asc`

Reply:
Changes done